### PR TITLE
Revert "doc: Remove instructions on how to authenticate using account…

### PR DESCRIPTION
**Pull request by rtfpessoa**
… API Token (#256)"

This reverts commit f5550403ee0836b41c55712d692cfd97a65e14f3.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -115,11 +115,23 @@ After having coverage reports set up for your repository, you must use Codacy Co
 1.  Set up an API token to allow Codacy Coverage Reporter to authenticate on Codacy.
     {: id="authenticate"}
 
-    Obtain the [project API Token](../codacy-api/api-tokens/#project-api-tokens) from the page **Integrations** in your Codacy repository settings. Then, set the following environment variable to specify your project API Token:
+    -   **If you're setting up coverage for one repository**, obtain the [project API Token](/repositories-configure/integrations/project-api/) from the page **Integrations** in your Codacy repository settings.
```
Comment (mrfyda) [inline docs/index.md:118]:
```suggestion
    -   **If you're setting up coverage for one repository**, obtain the [project API Token](../codacy-api/api-tokens/#project-api-tokens) from the page **Integrations** in your Codacy repository settings.
```

File: docs/index.md [hunk 2]
```diff
@@ -115,11 +115,23 @@ After having coverage reports set up for your repository, you must use Codacy Co
...
-    ```bash
-    export CODACY_PROJECT_TOKEN=<your project API Token>
-    ```
+        Then, set the following environment variable to specify your project API Token:
+
+        ```bash
+        export CODACY_PROJECT_TOKEN=<your project API Token>
+        ```
+
+    -   **If you're setting up and automating coverage for multiple repositories**, obtain an [account API Token](https://docs.codacy.com/related-tools/api-tokens/) from the page **Access management** in your Codacy account settings.
```
Comment (mrfyda) [inline docs/index.md:126]:
```suggestion
    -   **If you're setting up and automating coverage for multiple repositories**, obtain an [account API Token](../codacy-api/api-tokens/#account-api-tokens) from the page **Access management** in your Codacy account settings.
```

File: docs/index.md [hunk 3]
```diff
@@ -115,11 +115,23 @@ After having coverage reports set up for your repository, you must use Codacy Co
...
+
+        Then, set the following environment variables to specify the account API Token, the username associated with the account API token, and the repository for which you're uploading the coverage information:
+
+        ```bash
+        export CODACY_API_TOKEN=<your account API Token>
+        export CODACY_USERNAME=<your account username>
```
Comment (mrfyda) [inline docs/index.md:132]:
```suggestion
        export ???=<your account provider>
        export CODACY_USERNAME=<your account username>
```

**Review by mrfyda — APPROVED** [review]

**Comment by prcr** [comment]
Great stuff! :+1: :rocket:

**Pull request merged**